lib/Cleantalk/ApbctWP/Firewall/die_page_sfw.html

```diff
@@ -241,6 +241,23 @@ <h3>{SFW_DIE_CLICK_TO_PASS}</h3>
 		document.getElementById('js_passed').style.display = 'block';
 		document.getElementById('curr_date').innerHTML = ct_date.toGMTString();
 
+		// Add IP to Database with Ajax
+		jQuery(document).ready(function ()
+		{
+			const data = {
+				action: 'apbct_add_ip_to_db_action',
+				nonce: ctPublicFunctions._ajax_nonce,
+				ip: '{REAL_IP}'
+			};
+
+			jQuery.ajax({
```

Comment:
Should we do this on Anti-Crawler or Anti-Flood screens?

Reply:
I do not think so